safe/gis/vector/tools.py

```diff
@@ -302,15 +302,24 @@ def read_dynamic_inasafe_field(inasafe_fields, dynamic_field):
     :param dynamic_field: The dynamic field to use.
     :type dynamic_field: safe.definitions.fields
 
+    :param black_list: A list of fields which are conflicting with the dynamic
```

Comment:
Honestly, I am getting lost of these fields (especially the new one). Perhaps, we can make a diagram or something to make people easy to understand.

Reply:
True. I don't like this blacklist hack too but now I don't see any quick solution. It's getting complex day after day